recipes/rb-json/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "json" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 9dd1437156773f72c096058ec837faac1b00077121a3fd574e68f895ea3aa96b
+
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }} clean
```

Comment:
This gem extensions in sub-directories to ext that need to be cleaned up. You can see these from the `patchelf: wrong ELF type` warnings in the [linux build logs][linuxlogs].
<details>
  <summary>build logs snippet: lines [616,625]</summary>

```
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
patchelf: wrong ELF type
patchelf: --print-rpath failed for /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/ruby/gems/2.6.0/gems/json-2.2.0/ext/json/ext/parser/parser.o

Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
Warning: rpath /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_build_env/lib is outside prefix /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla (removing it)
patchelf: wrong ELF type
patchelf: --print-rpath failed for /home/conda/feedstock_root/build_artifacts/rb-json_1567546535860/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/ruby/gems/2.6.0/gems/json-2.2.0/ext/json/ext/generator/generator.o
```
</details>

```suggestion
    {% for dir in [ '', 'ext/generator', 'ext/parser' ] %}
    - make -C {{ PREFIX }}/lib/ruby/gems/{{ ruby }}.0/gems/{{ name }}-{{ version }}/ext/{{ name }}/{{ dir }} clean
    {% endfor%}    
```

[linuxlogs]: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=68987&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&taskId=bef775d0-a1a2-55a3-9721-17147029baf0&lineStart=616&lineEnd=625&colStart=1&colEnd=1